coilsnake/ui/cli.py

```diff
@@ -72,14 +89,46 @@ def _upgrade(args):
                     project_path=args.project_directory)
 
 def _scriptdump(args):
-	decompile_script(rom_filename=args.rom_filename, 
-					 project_path=args.project_directory)
+    decompile_script(rom_filename=args.rom_filename, 
+                     project_path=args.project_directory)
 
 def _patchrom(args):
-	patch_rom(clean_rom_filename=args.clean_rom,
-			  patched_rom_filename=args.output_rom,
-			  patch_filename=args.patch,
-			  headered=args.headered)
+    if args.headered == "true":
```

Comment:
i'm confused as to what you are telling me to do. I tried `header = args.headered == "true"` and i tried `header == args.headered == "true"` and both yielded an error. but i tested the patchrom command with an ips patch both with the false argument and the true argument, and tested the GUI the same way, and they both did the exact same thing. could you please clarify what you're trying to do?